build.sbt

```diff
@@ -4,7 +4,6 @@ import uk.gov.hmrc.DefaultBuildSettings.{defaultSettings, scalaSettings}
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
```

Comment:
Please remove unused import, thanks.

Reply:
Also in this file, can you please set the value [here](https://github.com/hmrc/lisa-api/blob/DDCE-3468/build.sbt#L28) to true